### DISCUSSION TITLE
Version control `edgedb.toml`

### DISCUSSION THREAD

**Discussion by notramo** [geldata/gel#5482] [Q&A]
Do I need to track `edgedb.toml` in version control? If yes, then please explain why.

**Comment by tailhook** [comment]
The most important thing in edgedb.toml right now is `edgedb.server-version`. So in simple words, this file in version control documents which version of EdgeDB current version of your app is tested with.

I can elaborate more on versioning and risks of not keeping EdgeDB version in the version control if you wish.

@raddevon, do we have something about that in the documentation?
